entrypoint.sh

```diff
@@ -320,23 +334,23 @@ echo "$INPUT_IDE_VERSIONS" | while read -r IDE_VERSION; do
   ZIP_FILE_PATH="$HOME/$IDE-$VERSION.zip"
 
   echo "Downloading $IDE $IDE_VERSION from [$DOWNLOAD_URL] into [$ZIP_FILE_PATH]..."
-  CURL_RESP=$(curl -L --silent --show-error -w 'HTTP/%{http_code} - content-type: %{content_type}' --output "$ZIP_FILE_PATH" "$DOWNLOAD_URL")
+  
+  CURL_RESP=$(curl -L --silent --show-error -w '%{json}' --output "$ZIP_FILE_PATH" "$DOWNLOAD_URL")
+  http_code==$(echo "${CURL_RESP}" | jq -r '.response_code // empty')
+  content_type==$(echo "${CURL_RESP}" | jq -r '.content_type // empty')
```

Comment:
```suggestion
  http_code=$(echo "${CURL_RESP}" | jq -r '.response_code // empty')
  content_type=$(echo "${CURL_RESP}" | jq -r '.content_type // empty')
```

you're getting an extra `=` prepended to these two vars.

```
$ > cat test.sh 
#!/bin/bash


CURL_RESP=$(curl -L --silent --show-error -w '%{json}' --output "ideaIC-2022.2.zip" https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/idea/ideaIC/2022.2/ideaIC-2022.2.zip)

echo $CURL_RESP

http_code==$(echo "${CURL_RESP}" | jq -r '.response_code // empty')
content_type==$(echo "${CURL_RESP}" | jq -r '.content_type // empty')

echo "${http_code} - ${content_type}"
```

output:
```
..."curl_version":"libcurl/8.1.2 (SecureTransport) LibreSSL/3.3.6 zlib/1.2.11 nghttp2/1.51.0"}
=200 - =application/octet-stream
```